internal/gapicgen/generator/genproto.go

```diff
@@ -132,9 +138,15 @@ func (g *GenprotoGenerator) Regen(ctx context.Context) error {
 		fn := fileNames
 		grp.Go(func() error {
 			log.Println("running protoc on", pk)
-			return g.protoc(fn)
+			return g.protoc(fn, true /* grpc */)
 		})
 	}
+	// TODO(noahdietz): This needs to be generalized to support any proto in googleapis-discovery.
```

Comment:
We can change the format of the file to have several commits to track if necessary as this is all hand-rolled(no pattern to follow).

Reply:
Yeah, I think we may have to eventually. I'm not sure the googleapis-discovery pipeline is fully automated yet, so it's not urgent.